### ISSUE TITLE
Switching pages after scrolled - end up at the top

### ISSUE THREAD

**Issue by Wolfr** [roxiness/routify#31]
If you switch pages on a website, you would expect to end up at the top (scroll position 0). This is currently not the case, the scroll position remains.

**Comment by Wolfr** [comment]
Update - I fixed this with

````
    afterUpdate(() => {
        window.scrollTo(0, 0);
    });
````

Doesn't feel very clean but it works.

I wonder if setting scroll to 0,0 is part of other routing projects 🤔

**Comment by jakobrosenberg** [comment]
You make a good point.

How about an afterNavigate hook?

```<!-- src/App.svelte -->
<script>
  import { Router, afterNavigate } from "@sveltech/routify";
  import { routes } from "@sveltech/routify/tmp/routes";

  afterNavigate = () => window.scrollTo(0, 0) 
</script>

<Router {routes} />
```

It could also be a runtime option, though obviously less transparent and powerful.

**Comment by Wolfr** [comment]
I don't see a reason why you wouldn't just lock it down in the framework. If the author wants to retain scroll position (can't come up with an example for this but hey, maybe someone else can), they could enable it. So `0,0` is the default; but you could opt out of that functionality if you have a reason to with a flag?

**Comment by jakobrosenberg** [comment]
I guess we could create an autoScrollToTop and set it to true by default. This way, if anyone is so inclined, they have the option to opt out.

**Comment by jakobrosenberg** [comment]
Should be fixed in the latest release.

**Comment by AlexanderBelkevich** [comment]
@jakobrosenberg  This feature doesn't work on 1.0.1-53.
Maybe, I need to add something in my code?

**Comment by jakobrosenberg** [comment]
@AlexanderBelkevich I'll have a look and get back to you.

**Comment by jakobrosenberg** [comment]
@AlexanderBelkevich This has been fixed in 1.0.1-54.

**Comment by AlexanderBelkevich** [comment]
@jakobrosenberg Awesome! Now it's working :) Thank you)
